src/handlers/handleCLIDownloader.ts

```diff
@@ -252,6 +313,37 @@ class DependenciesDownloader {
             logger.debug("CBShell is already installed");
             this.setToolActive(ToolStatus.AVAILABLE, shellPath, shell);
         }
+
+        const cbMigrate = downloads.get(this.TOOL_MDB_MIGRATE);
```

Comment:
Can we move all these in their respective functions?

Even do it with other tools, As it has become kinda hard t read and understand now

Reply:
Moved all to a seperate function for each.